### PR TITLE
Update variables names - Closes #1373

### LINKED ISSUE #1373: Fix inconsistent variable names in guide
**Describe the bug**
The code examples use incorrect variable names:

blockBuffer -> blockObject
encodedBlockAsBuffer -> encodedBlockObject

txBuffer ->  txObject
encodedTxAsBuffer -> encodedTxObject

**To Reproduce**
https://lisk.com/documentation/lisk-sdk/guides/app-development/decoding-transactions-blocks.html#decoding-encoding-blocks-and-their-included-transactions

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1373

### CODE REVIEW

File: modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_references_apiclient: references/lisk-elements/api-client.adoc
 :url_guides_config: guides/app-development/configuration.adoc#rpc
 
-The API of a blockchain node typically returns all requested transactions and blocks encoded as a Buffer.
+The API of a blockchain node typically returns all requested transactions and blocks encoded.
```
Comment (chris529) [inline modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc:17]:
```suggestion
The API of a blockchain node typically returns all requested transactions and encoded blocks.
```

**Review by chris529 — COMMENTED** [review]
Looks good, only 1 minor point 👍

File: modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc:17]: The meaning is more, that the API returns all blocks and transactions in encoded format.

Comment (chris529) [reply modules/ROOT/pages/guides/app-development/decoding-transactions-blocks.adoc:17]: Ah ha...Understood, OK then in this case it is fine as you had it originally. 👍

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Pull request merged**